pkgs/tools/misc/bunnyfetch/default.nix

```diff
@@ -1,28 +1,31 @@
-{ buildGoModule, fetchFromGitHub, lib }:
+{ buildGoModule, buildPackages, fetchFromGitHub, lib }:
 
-buildGoModule rec {
-  pname = "bunnyfetch";
-  version = "unstable-2021-05-24";
+let buildGoModule116 = buildGoModule.override {
+  go = buildPackages.go_1_16;
+};
+in
+  buildGoModule rec {
```

Comment:
`buildGoModule` already defaults to Go 1.16, so this isnt necessary.

Reply:
When I run

```
$ nix-build -E 'with import <nixpkgs> {}; callPackage ./default.nix {}'
```

Without overriding the version of Go I get this:

```

these derivations will be built:
  /nix/store/hvxz4ckckmwdr5kd7bjwazy5jzicp4a2-bunnyfetch-24370338b936bae1ebdefea73e8372ac0b4d2858.drv
building '/nix/store/hvxz4ckckmwdr5kd7bjwazy5jzicp4a2-bunnyfetch-24370338b936bae1ebdefea73e8372ac0b4d2858.drv'...
unpacking sources
unpacking source archive /nix/store/y2z2pwjcvwiav4n3mi45vmpbfraycazm-source
source root is source
patching sources
configuring
building
Building subPackage .
github.com/joho/godotenv
github.com/spf13/pflag
github.com/spf13/cobra
github.com/Rosettea/bunnyfetch/cmd
# github.com/Rosettea/bunnyfetch/cmd
cmd/info_linux.go:17:16: undefined: os.ReadFile
note: module requires Go 1.16
builder for '/nix/store/hvxz4ckckmwdr5kd7bjwazy5jzicp4a2-bunnyfetch-24370338b936bae1ebdefea73e8372ac0b4d2858.drv' failed with exit code 1
error: build of '/nix/store/hvxz4ckckmwdr5kd7bjwazy5jzicp4a2-bunnyfetch-24370338b936bae1ebdefea73e8372ac0b4d2858.drv' failed
```

However, when I override the version of Go (like in the PR) it build successfully. Is there something I'm missing with the requirements?

Reply:
How I built it is from my local copy of nixpkgs,
```
nixpkgs on  master took 12s ^ nix-build -A bunnyfetch
this path will be fetched (0.99 MiB download, 3.37 MiB unpacked):
  /nix/store/p1snkhk36f5as3psgiqh1agkmv3d20jr-bunnyfetch-unstable-2021-05-24
copying path '/nix/store/p1snkhk36f5as3psgiqh1agkmv3d20jr-bunnyfetch-unstable-2021-05-24' from 'https://cache.nixos.org'...
/nix/store/p1snkhk36f5as3psgiqh1agkmv3d20jr-bunnyfetch-unstable-2021-05-24

nixpkgs on  master took 37s ^ /nix/store/p1snkhk36f5as3psgiqh1agkmv3d20jr-bunnyfetch-unstable-2021-05-24/bin/bunnyfetch
            fortuneteller2k@superfluous
            OS NixOS
   (\ /)    Kernel 5.12.12-xanmod1-cacule
   ( . .)   Shell zsh
   c(")(")  WM wlroots wm
```